dashboard/app/views/levels/instructions.haml

```diff
@@ -0,0 +1,26 @@
+-#
+  This is a view for rendering a summary of all of the levels/instructions in a given
+  script, that should only be visible in levelbuilder mode
+
+- stages = local_assigns[:stages]
+
+- stages.each_with_index do |stage, index|
+  %h1{:style => "margin-top: 30px"}= "Stage #{(index + 1)}: #{stage.name}"
```

Comment:
Looks like this is the preferred style, so I should probably follow it. Does seem a little bit odds with our React approach (where we like having all the styling inlined and in the same file), but it's somewhat reasonable to have different preferences here.